### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by bkmceh**
@danmory @ezio_99 please check

### CODE REVIEW

File: app_java/src/main/java/ru/bkmceh/lab1/Controller.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    private static final String TIME_PATTERN = "yyyy/MM/dd HH:mm:ss";
+
+    @GetMapping
+    public String getTime() {
+        DateTimeFormatter timeColonFormatter = DateTimeFormatter.ofPattern(TIME_PATTERN);
+        return wrapTime(timeColonFormatter.format(LocalDateTime.now(ZoneId.of("UTC+3"))));
+    }
+
+    private String wrapTime(final String time) {
+        return """
```
Comment (danmory) [inline app_java/src/main/java/ru/bkmceh/lab1/Controller.java]: why view is in the controller?

File: app_java/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Moscow time checker
+
+## 💾 Information
+Sometimes you want to know time in Moscow, but you cant.\
+Now you can easily run this app, go to http://localhost:8080 and see Moscow time!
+
+## 📝 How to run
+### Main dependencies
```
Comment (danmory) [inline app_java/README.md:10]: Did you use linter?

**Comment by danmory** [comment]
You forgot about files JAVA.md and PYTHON.md with best practices applied

File: app_java/README.md (hunk 2 above)
Comment (ezio-42) [reply app_java/README.md:10]: can't

File: app_java/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Java - 15 or more
+- Maven - 3.6.3 or more
+### Building
+Go to ``app_java`` directory\
```
Comment (ezio-42) [inline app_java/README.md:12]: directory

File: app_java/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Moscow time checker
+
+## 💾 Information
+Sometimes you want to know time in Moscow, but you cant.\
+Now you can easily run this app, go to http://localhost:8080 and see Moscow time!
+
+## 📝 How to run
+### Main dependencies
+- Java - 15 or more
+- Maven - 3.6.3 or more
```
Comment (ezio-42) [inline app_java/README.md]: or higher

**Pull request closed without merging**